website/docs/quickstart.mdx

```diff
@@ -6,75 +6,68 @@ import Tabs from '@theme/Tabs';
-
-Request a license by signing up for a self-hosted trial [here](https://www.getunleash.io/pricing). Once you've signed up, you'll receive an email from Unleash containing your trial license key.
+![The Unleash Admin UI onboarding experience](/img/onboarding-experience.png)
 
-In the Admin UI, go to **Admin settings > Instance config > License**, copy the license key you received by email and click **Update license key**.
+If you prefer to manage your own infrastructure, see the [self-hosted setup guide](./using-unleash/deploy/getting-started). To use the open-source version, see the instructions in the [Unleash GitHub repository](https://github.com/Unleash/unleash?tab=readme-ov-file#get-started-with-unleash).
 
-The top banner now displays the number of days you have left on your free trial.
+## Sign up for an Unleash Enterprise Cloud trial
 
-![Unleash Admin UI log in screen](/img/quickstart-license.png)
+Start by signing up for a [free 14-day trial](https://www.getunleash.io/plans/enterprise-payg) of Unleash Enterprise Cloud.
+After you submit the form, you will receive a confirmation email. Follow the link in the email to set your password and log in to your Unleash instance.
 
+## Create your first feature flag
 
-## Create your first flag
+Once you've logged in, it's time to create your first feature flag:
 
-To create your first flag:
-1. Open the **Default** project.
+1. In the Unleash Admin UI, open the **Default** project.
 2. Click **New feature flag**.
 3. Enter a name, and click **Create feature flag**.
 
 For more details on creating feature flags, see [How to create a feature flag](/how-to-create-feature-flag).
 
-## Connect an SDK
+## Connect your application to Unleash
 
-Next, use one of the client or server-side [SDKs](/reference/sdks) to connect Unleash with your application.
+Next, use one of the client or server-side [SDKs](/reference/sdks) to connect Unleash with your application. 
 
+<details>
+  <summary>Need an example application to test with?</summary>
+
+    Explore our SDK examples directly in your browser with CodeSandbox. Check out the [Unleash SDK Examples](https://github.com/Unleash/unleash-sdk-examples) repository to try one now.
+</details>
 
 <Tabs groupId="connect-sdk-quickstart">
 
 <TabItem value="sdk-client-side" label="Connect a client-side SDK">
 
-1. Create a [frontend API token](/reference/api-tokens-and-client-keys#frontend-tokens). 
-2. Determine your Unleash URL, for example: `http://localhost:4242/api/frontend`. 
-3. Use the SDK to connect to Unleash in your application.
+1. **Create an API token**
+
+    In the Unleash Admin UI, create a [frontend API token](/reference/api-tokens-and-client-keys#frontend-tokens) in **Admin settings > Access control > API access**. 
+2. **Get your Unleash API URL**
+
+    Find the base URL of your instance (for example, `https://us.app.unleash-hosted.com/some-instance-id`). Your frontend API URL is this base URL with `/api/frontend` appended to it.
```

Comment:
```suggestion
    Find the base URL of your Unleash Cloud instance (for example, `https://us.app.unleash-hosted.com/some-instance-id`). Your frontend API URL is this base URL with `/api/frontend` appended to it.
```

Reply:
In the server-side SDK example, we call it the Unleash Cloud instance which I like. So adding that here.